duecredit/entries.py

```diff
@@ -8,51 +8,55 @@
 #
 # ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
 import re
+from typing import Optional
 
 import logging
 lgr = logging.getLogger('duecredit.entries')
 
 
 class DueCreditEntry:
-    def __init__(self, rawentry, key=None):
+    _key : str
+
+    def __init__(self, rawentry:str, key: Optional[str] = None) -> None:
         self._rawentry = rawentry
         self._key = key or rawentry.lower()
 
-    def __eq__(self, other):
+    def __eq__(self, other: object) -> bool:
```

Comment:
Change `object` to `Any` (Why is there no "Add a suggestion" button?)

Alternatively, change `object` to `DueCreditEntry` and get rid of the runtime check.

Reply:
__eq__ is replacing __cmp__ which has been dead code since Py3.